bin/main.rb

```diff
@@ -1,2 +1,28 @@
 #!/usr/bin/env ruby
-puts "Hello World!"
\ No newline at end of file
+
+  def board 
+    puts " 1 | 2 | 3 "
+    puts "-----------"
+    puts " 4 | 5 | 6 "
+    puts "-----------"
+    puts " 7 | 8 | 9 "
+  end
+
+puts "Enter Player-1 name"
+player_1 = gets.chomp
+puts "Enter Player-2 name"
+player_2 = gets.chomp
+
+puts "#{player_1} turn"
+puts "Pick any avaliable number 1 to 9"
+board
+player_1_choice = gets.chomp
+
+puts "#{player_2} turn"
+puts "Pick any avaliable number 1 to 9"
+board
+player_2_choice = gets.chomp
```

Comment:
This is a good place to use a loop since the same actions are repeated for each player until the game terminating conditions are met.